packages/tools/src/utilities/segmentation/InterpolationManager/updateRelatedAnnotations.ts

```diff
@@ -0,0 +1,30 @@
+import interpolate from '../../contours/interpolation/interpolate';
+import type { InterpolationViewportData } from '../../../types/InterpolationTypes';
+import deleteRelatedAnnotations from './deleteRelatedAnnotations';
+
+/**
+ * updatedRelatedContours - Update the same roi contour id contours on modifying
+ * one of them .
+ *
+ * @param eventData - Object.
+ * @returns null
+ */
+export default function updateRelatedAnnotations(
+  viewportData: InterpolationViewportData,
+  isLabelUpdate = true
+) {
+  const { annotation } = viewportData;
+
+  if (isLabelUpdate) {
+    console.log('isLabelUpdate');
```

Comment:
remove console log

Also I'm not sure what isLabelUpdate mean, I know flywheel was using it to create a "Segment" but we have moved away with that construct to a more sophisticated contourSegmentation